externs/browser/w3c_audio.js

```diff
@@ -26,245 +26,305 @@
 /**
  * @constructor
  */
-function AudioContext() {}
+function BaseAudioContext() {};
 
-/** @type {!AudioDestinationNode} */
-AudioContext.prototype.destination;
+/**
+ * @type {number}
+ * @const
+ */
+BaseAudioContext.prototype.currentTime;
```

Comment:
I appreciate that you want to define things in a logical order and alphabetize these properties.
That's a good idea to make finding them easier for the reader.
Unfortunately doing it now makes it harder to review the functional changes.

If you like, you could do a separate PR that just reorders things, then rebase this PR onto that.
That should be a super easy PR to approve.

If you prefer not to do that, please reorder these in a way that minimizes the differences to review.
Thanks.